worlds/alttp/Options.py

```diff
@@ -198,6 +198,27 @@ class EnemyShuffle(Toggle):
     display_name = "Enemy Shuffle"
 
 
+class PlandoEnemies(OptionList):
+    """Define a list of enemies to be placed where possible. Where they cannot be placed will be randomized enemies.
+    Only enabled when enemy shuffle is enabled. For valid enemies see
+    https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/alttp/Options.py#L201"""
```

Comment:
If you set the default value for the option to a list of all enemies, then the generated yaml template will come with a list of all of them as the default